pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/steps/CredentialWrapper.java

```diff
@@ -45,7 +45,7 @@
     private final String credentialId;
     private final List<Map<String, Object>> withCredentialsParameters;
 
-    CredentialWrapper(String credentialId, List<Map<String, Object>> withCredentialsParameters) {
+    public CredentialWrapper(String credentialId, List<Map<String, Object>> withCredentialsParameters) {
```

Comment:
`@Restricted(NoExternalUse.class)` since this will be refactored a bit later and you now made it `public`.